skyportal/tests/frontend/test_groups.py

```diff
@@ -99,7 +99,9 @@ def test_add_new_group_user_nonadmin(
     )
 
 
-@pytest.mark.flaky(reruns=2)
+@pytest.mark.xfail(
+    strict=False
```

Comment:
Can you check for sendgrid conf files at the toplevel of the module, then mark `skipif` if they are not there, rather than always xfailing?

Reply:
Sure :+1:

Reply:
@dannygoldstein what I ended up doing here is just checking whether invitations are enabled, and if so, checking for the corresponding notification (which now only is displayed if invites are enabled), and adding new users works as it did before when invites are disabled. No xfailing needed anymore at all.